src/components/mobile/Page.vue

```diff
@@ -9,7 +9,9 @@
       v-bind="$attrs"
       padded
       v-on="$listeners"
-    />
+    >
+      {{ ' ' }} <slot name="title" />
```

Comment:
`title` property and `title` slot shouldn't be used at the same time, so please remove `{{ ' ' }} `